src/index.js

```diff
@@ -275,7 +275,7 @@ export default class Tooltip {
     this.tooltipInput.placeholder = this.placeholder;
     this.tooltipInput.classList.add(this.api.styles.input);
     this.tooltipInput.classList.add(this.CSS.input);
-    if (this.spanTooltip) {
+    if (this.spanTooltip && this.spanTooltip.dataset.tooltip) {
```

Comment:
@MarioRodriguezS  Please review if we could refactor this way
```suggestion
    if (this.spanTooltip?.dataset?.tooltip) {
```